bin/parseDocs.ts

```diff
@@ -195,12 +195,27 @@ for (const key in json.recipes) {
 }
 
 // attach extractable resources instead of keeping empty array with "everything allowed"
-Object.keys(json.miners)
-	.filter((minerKey: string) => 0 === json.miners[minerKey].allowedResources.length)
-	.forEach((minerKey: string) => {
-		json.miners[minerKey].allowedResources =  Object
-			.keys(json.resources)
-			.filter((resourceKey: string) => json.items[resourceKey] && json.miners[minerKey].allowLiquids === json.items[resourceKey].liquid);
-	});
+for (const minerKey in json.miners) {
+	if (json.miners[minerKey].allowedResources.length > 0) {
+		continue;
+	}
+
+	const allowedResources = [] as string[];
+	for (const resourceKey in json.resources) {
+		if (!json.items[resourceKey]) {
+			throw new Error(`Item of resource type "${resourceKey}" was not found.`);
```

Comment:
Probably shouldn't throw here. I can't imagine you really want to stop execution for this issue. I assume you were just curious if there happened to be any missing items and forgot to remove this before committing for PR?

Reply:
@jshepler this is inside a CLI script that parses the file. This error tells me that there's a resource that doesn't have it's item data exported, so there's probably a bug either in my script or in the Docs.json file. This code is never run in a browser, so it's completely fine (and also pretty much required) to throw an error here.

Reply:
Cool, my apologies.